blog/_posts/2021-01-11-scala-native-0.4-release.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: blog-detail
+post-type: blog
+by: Wojciech Mazur, Scala Center
+title: A sneak peek into upcoming Scala Native 0.4 release
+---
+
+If you ever thought about running your perfect Scala code in an environment other than JVM, you've probably already met
+Scala Native---a project with a promise of instant startup, low memory usage, interop with C libraries, and competitive performance
+thanks to [Interflow optimizer](https://scala-native.readthedocs.io/en/latest/blog/interflow.html).
+
+However, since there have been no release in over a year, you may ask yourself---is this project even alive?<br>
+With the long-awaited release it proves it is! Today we are going to discuss the most impacting new features and changes 
+coming in this new major release. We will start with changes according to Scala Native compiler plugin, especially supported Scala versions and cross-platform testing.
+Next we will talk about changes to C interoperability and SBT build that need to be taken into account since when working this release.
+Last, but not least we will discuss why we needed to break binary compatibility with previous versions.
+
+
+## Support for latest Scala 2 versions
+Scala Native finally starts catching up with the rest of the Scala ecosystem, because until now there was only support for Scala 2.11,
+which is now perceived by most of the developers as deprecated. We can proudly announce that Scala Native is now 
+compatible with Scala versions up to 2.12 and 2.13.
+We also continue to support Scala 2.11 for which we have added new features and fixed known bugs.
+
+## Cross-platform testing using JUnit
+Scala Native now comes with JUnit support out of the box, this means that you can write tests in the same way you would do for a Java project.
```

Comment:
Why would I want to just JUnit instead of e.g. ScalaTest or MUnit? Must I use JUnit? (I think I know the answers, but I think some readers will be wondering.)